Dockerfile

```diff
@@ -0,0 +1,39 @@
+# FROM ubuntu-upstart:precise
+# http://phusion.github.io/baseimage-docker/
+FROM phusion/baseimage:0.9.9
+
+# Install pre-pre-requisites.
+RUN apt-get update
```

Comment:
Gotcha. When I learnt about Docker needing things to not be daemonized I figured that was probably a show stopper 😞